modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/dto/v1_0/util/ObjectFieldUtil.java

```diff
@@ -58,7 +58,10 @@ public static long getListTypeDefinitionId(
 
 		if (!StringUtil.equals(
 				objectField.getBusinessTypeAsString(),
-				ObjectFieldConstants.BUSINESS_TYPE_PICKLIST)) {
+				ObjectFieldConstants.BUSINESS_TYPE_PICKLIST) &&
+			!StringUtil.equals(
+				objectField.getBusinessTypeAsString(),
+				ObjectFieldConstants.BUSINESS_TYPE_MULTISELECT_PICKLIST)) {
```

Comment:
@p-albuquerque could you sort this alphabetically? i.e., `BUSINESS_TYPE_MULTISELECT_PICKLIST` before `BUSINESS_TYPE_PICKLIST`